LICENSE-binary

```diff
@@ -541,3 +571,67 @@ data/mllib/images/kittens/54893.jpg
 data/mllib/images/kittens/DP153539.jpg
 data/mllib/images/kittens/DP802813.jpg
 data/mllib/images/multi-channel/chr30.4.184.jpg
+
+
+Golang LICENSE derived from Russ Cox's RE2
```

Comment:
Sorry, I mean this needs to be a text file under licenses-binary/

Reply:
> Sorry, I mean this needs to be a text file under licenses-binary/

Add it, the file name is ok? for `re2j`, it is personal license follow go license. and for `dnsjava` also personal license.